tests/unit-tests/gconstruct/test_construct_graph.py

```diff
@@ -2725,3 +2728,543 @@ def test_two_none(idx_none_0, idx_none_1):
 
     test_two_none(0, 4)
     test_two_none(4, 10)
+
+def create_featless_data(graph_dir):
+    edge_dir = os.path.join(graph_dir, "edges")
```

Comment:
May we add some comments on why we are creating for both int and str type node here?